lib/esp/extensions/active_resource/formats/json_api_format.rb

```diff
@@ -43,7 +43,7 @@ def decode(json)
         Formats.remove_root(parse_json_api(ActiveSupport::JSON.decode(json)))
       end
 
-      private
+      private_class_method
```

Comment:
Apparently it does not do what I want.  Changing everywhere I've done that over the last 4 months now.